addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3344,6 +3344,11 @@ options.registry.WebsiteAnimate = options.Class.extend({
         await this._super(...arguments);
         this.isAnimatedText = this.$target.hasClass('o_animated_text');
         this.$optionsSection = this.$overlay.data('$optionsSection');
+        if (this.isAnimatedText) {
+            for (const handle of this.$overlay[0].querySelectorAll(".o_handle")) {
+                handle.style.pointerEvents = "none";
+            }
+        }
```

Comment:
Probably only a oneliner that way: `this.$overlay[0].classList.toggle('pe-none', this.isAnimatedText);` ?

Reply:
Code updated to apply the style on the `.o_handles` (because of [this code](https://github.com/odoo/odoo/blob/15.0/addons/web_editor/static/src/scss/wysiwyg_snippets.scss#L2009)), and I don't think `pe-none` is available before BS5... But I will update it to toggle the class starting from `16.0` :+1: